pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -64,6 +67,11 @@ const (
 // initConfigController creates the config controller in the pilotConfig.
 func (s *Server) initConfigController(args *PilotArgs) error {
 	meshConfig := s.environment.Mesh()
+	if args.Config.EnableConfigAnalysis {
+		if err := s.initInprocessAnalysisController(args); err != nil {
+			return err
+		}
+	}
```

Comment:
Move lines 70-74 to the else conditional at line 93 since CRD status depends on kube-apiserver.

Reply:
done.